io.github.randovania.Randovania.yml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.randovania.Randovania
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [ org.freedesktop.Sdk.Extension.mono6 ]
+finish-args:
+    - --device=dri          # UI will want hw acceleration
+    - --share=network       # Multiworld support
+    - --share=ipc           # Needed for X11 support
+    - --socket=wayland
+    - --socket=fallback-x11
+    - --filesystem=xdg-run/app/com.discordapp.Discord:create
+command: /app/share/randovania/randovania
```

Comment:
This is not a blocker, but we generally prefer an entrypoint in /app/bin. Can be done using a shell wrapper.

Reply:
Would this be okay for the shell wrapper?

```
#!/bin/sh
exec /app/share/randovania/randovania
```

Reply:
Probably should have `$@` too, but yeah, that sounds good.